### ISSUE TITLE
Problem during data loading on custom h5ad

### ISSUE THREAD

**Issue by ddemaeyer** [czbiohub-sf/excellxgene#15]
Loading internally generated h5ad into excellxgene results in error in the log:

```
ERROR:backend.server.app.app:Exception on /api/v0.2/layout/obs [GET]
TypeError: '(slice(None, None, None), slice(0, 2, None))' is an invalid key
```
The result in the browser is that there is an unexpected http response (see screenshot).

Currently, I did not yet add any of the data that generate the problem. Data runs fine in the cellxgene-1.0.1. 

![Screenshot 2022-06-20 at 13 37 55](https://user-images.githubusercontent.com/1774095/174593685-2bd47349-0f78-4387-a511-aca173e2ac79.png)

Versions affected:
- 2.8.1 
- 2.9.1

**Comment by lcdorman** [comment]
Were you able to fix this? I would check your layer keys from the object, and when I had an "invalid key" issue it was due to AnnData versions. Excellxgene runs with 0.7.8 and may fail to open objects built in 0.8.0.

**Comment by yangjun9095** [comment]
Closing this issue as we haven't heard back. That issue should be resolved in v2.9.6 release.
